setup.py

```diff
@@ -34,21 +36,42 @@ def run(self):
         # Jython plugin setuptools version 0.6...
         build_py.run(self)
 
+
 setup(name='pyoracc',
-      version='0.0.1',
+      version='0.1.0',
       author='James Hetherington',
       author_email='j.hetherington@ucl.ac.uk',
```

Comment:
Perhaps change this email to the team, since James is sadly leaving UCL in the next couple months. The team's email is `rc-softdev at ucl.ac.uk`.